src/renderer/pages/Basket/model/basket-page-model.ts

```diff
@@ -22,12 +23,31 @@ import {
 } from '@features/operations/OperationsValidation';
 import { signOperationsModel } from './sign-operations-model';
 
-const txSelected = createEvent<ID>();
+const txSelected = createEvent<{ id: ID; value: boolean }>();
 const txClicked = createEvent<BasketTransaction>();
 const allSelected = createEvent();
+const signStarted = createEvent();
+const validationCompleted = createEvent();
+const signContinued = createEvent();
+const signTransactionsReceived = createEvent<{
+  valid: BasketTransaction[];
+  invalid: BasketTransaction[];
+}>();
+const validationWarningShown = createEvent<{
+  valid: BasketTransaction[];
+  invalid: BasketTransaction[];
+}>();
+const proceedValidationWarning = createEvent<{
+  valid: BasketTransaction[];
+  invalid: BasketTransaction[];
+}>();
+const cancelValidationWarning = createEvent();
 
-const $selectedTxs = createStore<ID[]>([]);
+const $selectedTxs = createStore<Set<number>>(new Set());
```

Comment:
In this case `Set` is used for filtering uniq values, as I understood it. But for me using `Set` inside effector store look unusual. Also, it seems that logic of filtering in 2 different effects propagates and affects shape of storage. Maybe, you can write some helpers for manipulating array and use them inside those effects?